kits/daisy/src/main.cpp

```diff
@@ -0,0 +1,648 @@
+    mode_label->setFont(font);
+    mode = new QLabel("---");
+
+    QLabel* feedback_status_label = new QLabel("Module Ethernet Connection:");
+    font = feedback_status_label->font();
+    font.setWeight(QFont::Bold);
+    feedback_status_label->setFont(font);
+
+    feedback_status = new QLabel("Good");
+    QPalette palette = feedback_status->palette();
+    palette.setColor(QPalette::Background, Qt::green);
+    feedback_status->setAutoFillBackground(true);
+    feedback_status->setPalette(palette);
+
+    QLabel* frequency_label = new QLabel("Logging Frequencies (low/high):");
+    font = frequency_label->font();
+    font.setWeight(QFont::Bold);
+    frequency_label->setFont(font);
+    std::string frequency_text = std::to_string(params.low_log_frequency_hz_) + "/" + std::to_string(params.high_log_frequency_hz_);
+    QLabel* frequency = new QLabel(frequency_text.c_str());
+
+    vLayout->addWidget(mode_label);
+    vLayout->addWidget(mode);
+    vLayout->addWidget(feedback_status_label);
+    vLayout->addWidget(feedback_status);
+    vLayout->addWidget(frequency_label);
+    vLayout->addWidget(frequency);
+    ////////////////////////
+  }
+  else
+  {
+    QVBoxLayout *vLayout = new QVBoxLayout(widget);
+    QHBoxLayout *hLayout1 = new QHBoxLayout();
+    QHBoxLayout *hLayout2 = new QHBoxLayout();
+    QHBoxLayout *hLayout3 = new QHBoxLayout();
+    QLabel* mode_label = new QLabel("Current Mode:");
+    auto font = mode_label->font();
+    font.setWeight(QFont::Bold);
+    mode_label->setFont(font);
+    mode = new QLabel("---");
+
+    QLabel* feedback_status_label = new QLabel("Module Ethernet Connection:");
+    font = feedback_status_label->font();
+    font.setWeight(QFont::Bold);
+    feedback_status_label->setFont(font);
+    feedback_status = new QLabel("Good");
+    QPalette palette = feedback_status->palette();
+    palette.setColor(QPalette::Background, Qt::green);
+    feedback_status->setAutoFillBackground(true);
+    feedback_status->setPalette(palette);
+
+    QLabel* frequency_label = new QLabel("Logging Frequencies (low/high):");
+    font = frequency_label->font();
+    font.setWeight(QFont::Bold);
+    frequency_label->setFont(font);
+    std::string frequency_text = std::to_string(params.low_log_frequency_hz_) + "/" + std::to_string(params.high_log_frequency_hz_);
+    QLabel* frequency = new QLabel(frequency_text.c_str());
+
+    hLayout1->addWidget(mode_label);
+    hLayout1->addWidget(mode);
+    hLayout2->addWidget(feedback_status_label);
+    hLayout2->addWidget(feedback_status);
+    hLayout3->addWidget(frequency_label);
+    hLayout3->addWidget(frequency);
+    vLayout->addLayout(hLayout1);
+    vLayout->addLayout(hLayout2);
+    vLayout->addLayout(hLayout3);
+  }
+
+  widget->setWindowTitle(QStringLiteral("heXapod Control"));
+  widget->show();
+  widget->resize(overall_width, overall_height);
+
+  Eigen::Vector3f translation_velocity_cmd;
+  translation_velocity_cmd << 0, 0, 0;
```

Comment:
I really despise this syntax with Eigen. Why not just `setZero()`? This was a big pain point for me when I first starting using it, and the `,` operator is just pure evil

Reply:
Fixed.